alert/alert.css

```diff
@@ -0,0 +1,50 @@
+/* alert */
+.alert-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: space-between;
+  height: 15rem;
+}
+
+.alert {
+  width: 90%;
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  padding: 0.5rem 1rem;
+}
+
+.alert div {
+  margin-right: 1.5rem;
+  width: 2rem;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
```

Comment:
You can take all the class with repeating property like `display:flex, align-items:center` and instead of re-writing them, write them once for all class and avoid repeatation.
```
.alert, .alert div{ 
display: flex;
}
```